server/Makefile

```diff
@@ -21,12 +21,10 @@ binbase = \
 	pbench-base.sh
 
 # pbench-server-activate-create-crontab is needed by the ansible installation
-# playbook.  pbench-remote* and pbench-satellite-state-change.py are needed on
-# satellites only, but we install them everywhere.
+# playbook.  pbench-remote* are needed by pbench-sync-satellite.
```

Comment:
Where are the `pbench-remote-*` commands? They're not in either `bintools` or `binlinks`. Didn't we just review *removing* them in #3333??

Reply:
`server/bin`?

Reply:
I have not heard of `bintools` or `binlinks` before.

Reply:
Sorry: defined immediately below, as `bintools = pbench-trampoline [...]`, etc.

That is, so far as I can see there's no reference to `pbench-remote*` anywhere in this `Makefile` except this comment. And I believe that as of #3333 they no longer exist on the 0.69 branch. Certainly not in `server/bin`. 😕 

https://github.com/distributed-system-analysis/pbench/tree/b0.69/server/bin

Reply:
Bugger, thanks, I'll fix.

Reply:
Fixed.